README.md

```diff
@@ -1,2 +1,79 @@
 # djangocms-git-md-page
+
 DjangoCMS plugin for displaying a page with markdown in git repository.
+
+## Installation
+
+1. Add the `djangocms-git-md-plugin` into requirements and install it into the environment.
+
+```python
+# setup.py
+setup(
+    install_requires=[
+        # ...
+        'djangocms-git-md-page @ git+https://github.com/CZ-NIC/djangocms-git-md-page.git',
+    ]
+)
+```
+
+2. Add `git_md_page` into the `INSTALLED_APPS`
+
+```python
+# settings.py
+INSTALLED_APPS = [
+    # ...
+    'git_md_page',
+]
+```
+
+3. Create an entry in `urls` for incoming notifications from GitHub.
+
+```python
+# urls.py
+urlpatterns = [
+    # ...
+    url(r'^git_md_page/', include('git_md_page.urls')),
+]
+```
+
+4. Run migrations
+
+```shell
+$ python manage.py migrate
+```
+
+## Usage
+
+In the CMS administration, click the `+` symbol to add a new plugin. This plugin is named `Git MD page` and is located
+in the `Others` section.
+
+First of all, it is required to select the source repository. One can choose from the available choices or create a new
+choice, new source repository, by clicking the `+` symbol next to the select box. Every repository is defined by its
+address and branch. In the form, there are hints what value should be used. Also, there is another field called 
+`secret`. Its default value is a random string and this random value can be left. This `secret` value is then used for
```

Comment:
Personally, I don't really understand what `and this random value can be left` means.

Reply:
I changed it to `which can be left for further usage`